src/Workleap.DomainEventPropagation.Publishing/Workleap.DomainEventPropagation.Publishing.csproj

```diff
@@ -19,6 +19,10 @@
     <Description>Domain event propagation publisher. Includes built-in EventGrid client and list of topics. Contact the Core team for any questions.</Description>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <NoWarn />
+  </PropertyGroup>
+
```

Comment:
I would think we would get rid of the `NoWarn` at the top and the big comment.

Also because we are targeting .NET Standard 2.0, not all built-in Microsoft rules are enabled. We need to manually update them (and in any other .NET Standard 2.0 project) by adding this:

![image](https://github.com/gsoft-inc/wl-domain-event-propagation/assets/14242083/fc20e947-c01c-4c66-a421-37f19b495b07)

You'll start seeing warnings that would naturally appear in .NET 6+ projects